scripts/reset-all-repos.sh

```diff
@@ -36,18 +36,20 @@ fi
 if [ "$reset_to_main" = true ] ; then
     echo Resetting all repos to main
 
-    cd $workspace_dir
+    (
+      cd $workspace_dir
```

Comment:
What's the expected value of `workspace_dir`/`$1`? Is it `..` and we expect the script to be run from the top level of this repo/dir? It works obviously, but there's a slightly weird feeling dependency between the two `cd` commands that makes navigation feel a bit iffy. This subshell change doesn't really affect it positively or negatively mind, so probably don't need to change it here. Just flagging as it stood out as I was reading.